libs/react-helpers/src/lib/market-expires.tsx

```diff
@@ -36,12 +36,14 @@ export const getMarketExpiryDateFormatted = (
   return null;
 };
 
-export const getExpiryDate = (market: SingleMarketFieldsFragment): string => {
-  const metadataExpiryDate = getMarketExpiryDate(
-    market.tradableInstrument.instrument.metadata.tags
-  );
-  const marketTimestampCloseDate =
-    market.marketTimestamps.close && new Date(market.marketTimestamps.close);
+export const getExpiryDate = (
+  //   market: SingleMarketFieldsFragment,
```

Comment:
why this field is commented?

Reply:
Oh that's an oversight, sorry - I'll fix!

Reply:
seems like all params of this method are optional. where can it get the data from?

Reply:
It should have been removed when I expanded the arguments the function expects.

Reply:
> seems like all params of this method are optional. where can it get the data from?

This is a good point - I wanted to avoid refactoring too much of the internals of this function as well as the signature, but I think you're right it doesn't make sense to have all these optional params. I'll rework this too.

Reply:
OK I've updated this now @MadalinaRaicu if you want to have another look!